cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -89,13 +89,16 @@
     private final String httpsProxyTunnel;
     private final String authorization;
 
+    /**
+     * @deprecated Use {@link #CLI(hudson.cli.CLIConnectionFactory)
+     */
+    @Deprecated
     public CLI(URL jenkins) throws IOException, InterruptedException {
         this(jenkins,null);
     }
 
     /**
-     * @deprecated
-     *      Use {@link CLIConnectionFactory} to create {@link CLI}
+     * @deprecated Use {@link #CLI(hudson.cli.CLIConnectionFactory)
```

Comment:
It's wrong change. Idea was to use `CLIConnectionFactory.connect()` that calls package constructor while deprecated constructors calling them instead `CLIConnectionFactory`.